src/snowcli/cli/snowpark/function/commands.py

```diff
@@ -149,7 +172,7 @@ def upload_snowpark_artifact(
     return artifact_file
 
 
-@app.command("update")
+@app.command("update", hidden=True)
```

Comment:
Why hidden?

Reply:
To preserve the logic but hide the command. In near future it will be removed